### ISSUE TITLE
Possibility for AudioFocus release for the YouTubePlayerView

### ISSUE THREAD

**Issue by nicholaschum** [PierfrancescoSoffritti/android-youtube-player#691]
A loaded YouTube video is muted, but I would like to have the existing audioFocus be retained, so that when YouTubePlayerView starts playing, it would not pause the existing audio source, e.g. Spotify, or any other app.

This issue relates to #397 however I'm thinking more of how to get it programmatically functioning, whether it be a bypass or a workaround.

Thanks!

**Comment by PierfrancescoSoffritti** [comment]
As said in #397, i don't know how to do this. Let us know if you make progress :)

**Comment by nicholaschum** [comment]
I do see that the mobile version of YouTube on Google Chrome has this "Tap to Unmute" functionality, where it starts playing the YouTube video without hijacking the audioFocus when the YouTube page loads.

**Comment by nicholaschum** [comment]
@PierfrancescoSoffritti I have found a few methods of executing this requestAudioFocus() removal, however it is merged into my custom fork of this library.

https://github.com/nicholaschum/android-youtube-player

In general, we must add `autoplay: 1`, `mute: 1` into the playerVars in `ayp_youtube_player.html`. This would allow the IFrameAPI loaded WebView to automatically play the video in muted mode.

My library already has implemented hooks so that I can call it and it will dynamically load it into `<<injectedPlayerVars>>`.

Next, I have also modified the `ayp_youtube_player.html`, under events for `onYouTubeIframeAPIReady()`, I've set it to mute: ```onReady: function(event) {
                        event.target.mute();
                        YouTubePlayerBridge.sendReady();
                    },```
For library purposes, I could also create a commit that makes this perform the same way as `<<injectedPlayerVars>>`.

Also, from my testing, we must also ensure visibility of the YouTubePlayerView is set to `android:visibility="visible"` or else it would still try to requestAudioFocus(). I initially wasted a lot of time trying to set it to "gone" at first, and it kept stealing audio focus.

My fork is pretty much tailored for my app's purposes, but it includes other functions such as initializing the WebViewYouTubePlayer without focusable traits, e.g. `focusable = false` or `focusableInTouchMode = false` as I needed to make it so keyboard/DPAD functions won't touch the WebView objects, as well as adding other IFrameAPI playerVars.
